include/mbedtls/platform_util.h

```diff
@@ -25,7 +25,18 @@
 #ifndef MBEDTLS_PLATFORM_UTIL_H
 #define MBEDTLS_PLATFORM_UTIL_H
 
+#if !defined(MBEDTLS_CONFIG_FILE)
+#include "mbedtls/config.h"
+#else
+#include MBEDTLS_CONFIG_FILE
+#endif
+
+#include "mbedtls/platform_time.h"
```

Comment:
Shouldn't this header be included under a `MBEDTLS_HAVE_TIME_DATE` guard?

Reply:
I think we should move this include to the section guarded by `MBEDTLS_HAVE_TIME_DATE` below.

Reply:
Without the guard, this might lead to compilation failure if `MBEDTLS_HAVE_TIME` is not defined, because `mbedtls/platform_time.h` will include `time.h`.

Reply:
Addressed in 5a7fe14.